### ISSUE TITLE
New API Call `getAllAliasesOnSale`

### ISSUE THREAD

**Issue by ohager** [signum-network/signum-node#625]
At the moment, it's not easy to find out which aliases are on sale. Would be useful to have methods that returns all aliases which are on sale. As this can be a lot in the future the returned list must be limited by `firstIndex` and `lastIndex` like in other mass queries

A potential query may look like this

```sql
select * from "alias"
inner join "alias_offer" ao on "alias"."db_id" = ao."db_id"
```

**Comment by frankTheTank72** [comment]
![grafik](https://user-images.githubusercontent.com/39116221/181376986-e955c5da-6c22-40b7-beb8-4feb392c85e6.png)

We have a general sell to "all" and a sell to a given account - both should be returned

**Comment by ohager** [comment]
> ![grafik](https://user-images.githubusercontent.com/39116221/181376986-e955c5da-6c22-40b7-beb8-4feb392c85e6.png)
> 
> We have a general sell to "all" and a sell to a given account - both should be returned

Actually, having the `priceNQT` field in the response is the indicator for a running sale. Having the `buyer` means, it's just for one recipient. Regarding the response I do not see any need for an action.

**Comment by ohager** [comment]
> ![grafik](https://user-images.githubusercontent.com/39116221/181376986-e955c5da-6c22-40b7-beb8-4feb392c85e6.png)
> 
> We have a general sell to "all" and a sell to a given account - both should be returned

with the given query, both would be considered

**Comment by frankTheTank72** [comment]
yep - just wanted to point it out

**Comment by ohager** [comment]
fixed in #638
